From b8bf0b19ee743212e4745cebdd7975999d1ae2b5 Mon Sep 17 00:00:00 2001
From: boombatower <boombatower@google.com>
Date: Tue, 12 Nov 2013 15:30:09 -0800
Subject: [PATCH] Define MEMCACHE_COMPRESSED constant to work around lack of
 one on appengine.

To be fixed in upcoming release.
---
 memcache.inc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/memcache.inc b/memcache.inc
index 4dc5b07..16b8909 100644
--- a/memcache.inc
+++ b/memcache.inc
@@ -1,4 +1,5 @@
 <?php
+const MEMCACHE_COMPRESSED = 2;
 
 require_once dirname(__FILE__) . '/dmemcache.inc';
 
-- 
1.8.4.1
